crazyflie_cpp/include/crazyflie_cpp/Crazyflie.h

```diff
@@ -242,6 +264,7 @@ class Crazyflie
   template<typename T>
   friend class LogBlock;
   friend class LogBlockGeneric;
+  friend class CrazyflieROS;
```

Comment:
I was looking at how to implement this this morning and it occurred to me that the I added the friend class so that the CrazyflieRos object could call Crazyflie::sendPacket. This is a separate feature to what we discussed in the issue (we only discussed receiving packets from the crazyflie, but not transmitting them).

In this instance a callback does not make sense to me. What would be the job of the callback, and when would it be executed? What we want is for the CrazyflieROS object to provide the Crazyflie object with packet data, and ensure that at some point in time that Crazyflie::sendPacket is called with the packet data.

Perhaps something like the following would work: 

1) sendPacket service is called in a CrazyflieROS object. The packet stored in the service request is placed into a queue of outgoing packets within the Crazyflie object.

2) when sendPing is called, the Crazyflie object checks if there are outgoing packets to be transmitted, and then transmits them using Crazyflie::sendPacket, else it transmits a Ping.

This would remove the forward dependency of Crazyflie on CrazyflieROS, while preserving the protected status of Crazyflie::sendPacket.